.storybook/preview.js

```diff
@@ -1,2 +1,29 @@
-import "./stories.css"
-import "../assets/index.css"
+require("../assets/index.css")
+require("./stories.css")
+
+import { addParameters } from '@storybook/react';
+
+const screens = {
+  xxs: '375px',
+  xs: '480px',
+  sm: '576px',
+  md: '768px',
+  lg: '1024px',
+  xl: '1280px',
+  xxl: '1920px'
+}
```

Comment:
is it possible to require the `defaultConfig` and iterate over `tailwindConfig.theme.screens` instead of recreating the object here?

Reply:
Ohhh nice idea. Cleaning, cleaning 🧼 🧹 🧽

Reply:
If I remember correctly, by importing the tailwind config, we will increase the bundle size, that's why when we first added it, we also hardcoded the screen sizes

Reply:
correct, but this should be local to storybook and I would not care about its bundle size. if it's too much of a hassle, let's duplicate it. in the course of the esm build, I've also been thinking about just exposing the theme and not the full config, so no third parties can sneak in here. most of them are unused by now anyway